src/main/java/com/timelogr/enterprise/service/TimeLogrServices.java

```diff
@@ -0,0 +1,27 @@
+package com.timelogr.enterprise.service;
+import com.timelogr.enterprise.dto.Employee;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+* This class is currently testing Dummy Data
+ **/
+@Service
+public class TimeLogrServices implements ITimeLogrService {
+
+    private static final List<Employee> employeeList = new ArrayList<>();
+
+    static{
+        for(int i =0; i<10; i++){
+            employeeList.add(new Employee(i, "Human" + i,"d","d"));
+        }
+    }
+
+    @Override
+    public List<Employee> getAllEmployees() {
+        return employeeList;
+    }
+}
```

Comment:
Initially this class didn't use an interfaces, however I believe that using an interface could benefit your project in the future. Read more here (this is StackOverflow post may be beneficial in understanding why): [https://stackoverflow.com/questions/240152/why-would-i-want-to-use-interfaces](https://stackoverflow.com/questions/240152/why-would-i-want-to-use-interfaces)

Reply:
Good.